datalake/src/main/resources/application.yml

```diff
@@ -80,7 +80,7 @@ altus:
     enabled: true
     endpoint: localhost:8982
   datalakedr:
-    enabled: false
+    enabled: true
```

Comment:
this value change doesn't need to be committed.

Reply:
This configuration is used as below
1. CBD to decide if datalake-dr service should be started.
2. DR Client looks for this configuration to see if the dr-service is configured.

In the case of integration tests for resize, DR Client looks at this configuration. With this configuration set to false, backup/restore is not performed in the integration tests.

There is an overhead though. datalake-dr service is started by default in local development environments.

Reply:
Do we also need to make changes to CBD to go along with this new behavior?

Reply:
There are no changes needed. It's just that the datalake-dr service is started by default.

Reply:
removed it for now.